### PR TITLE
Styling the project

### PR DESCRIPTION
**Pull request by Said-laasri**
- followed design in [Zeplin](https://zeplin.io/).
- Used the information detailed in Zeplin to style the website.
- final design match Zeplin's design.
- Thanks  ==> **Usman Suleiman Babakolo**  For a great project design

### CODE REVIEW

File: src/components/Book.jsx [hunk 1]
```diff
@@ -19,19 +23,25 @@ function Book(props) {
...
         <div className="book-info">
+          <h3>{category}</h3>
           <h2>{title}</h2>
           <h3>{author}</h3>
-          <h3>{category}</h3>
         </div>
         <div className="btn-control">
-          <button type="button" onClick={bookRemove}>
+          <button className="btn-cl" type="button">Comments</button>
+          <button className="btn-cl" type="button" onClick={bookRemove}>
```
Comment (ahmetbozaci) [inline src/components/Book.jsx]: - [x] You almost finish the style. Kindly remove the background color and make the page fullscreen 
image
___
ORIGINAL
image

File: src/components/Book.scss [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+  gap: 5rem;
+}
+
+.book {
+  display: flex;
+  flex-direction: column;
+}
+
+.book-info {
+  display: flex;
```
Comment (ahmetbozaci) [inline src/components/Book.scss]: - [x] For DRY CSS you should combine duplicate properties. 
Example, you can create class `.d-flex { display: flex}` and use this class in HTML file.
_This is just one example. Check your CSS file and if some CSS rules are the same please combine them also._
- [More information about DRY principles](https://www.freecodecamp.org/news/7-important-tips-for-writing-better-css/) 

- [Html and CSS best practices in curriculum](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md#:~:text=If%20you%20want%20to%20create%20a%20CSS%20rule%20for%20multiple%20elements%2C%20use%20a%20class%20attribute%20instead.%20For%20example%2C%20instead%20of%20using%20color%3A%20%233498db%3B%203%20times)

File: src/components/Book.scss [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+  flex-direction: column;
+}
+
+h2 {
+  font-family: RobotoSlab, sans-serif;
```
Comment (ahmetbozaci) [inline src/components/Book.scss]: - [x] For DRY CSS you should combine duplicate properties. 
Example, you can create class `.d-flex { display: flex}` and use this class in HTML file.
_This is just one example. Check your CSS file and if some CSS rules are the same please combine them also._
- [More information about DRY principles](https://www.freecodecamp.org/news/7-important-tips-for-writing-better-css/) 

- [Html and CSS best practices in curriculum](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md#:~:text=If%20you%20want%20to%20create%20a%20CSS%20rule%20for%20multiple%20elements%2C%20use%20a%20class%20attribute%20instead.%20For%20example%2C%20instead%20of%20using%20color%3A%20%233498db%3B%203%20times)

File: src/components/Book.scss [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+  font-style: normal;
+  line-height: normal;
+  color: #121212;
+  margin: 0;
+}
+
+h3 {
+  opacity: 0.5;
+  font-family: Montserrat, sans-serif;
+  font-size: 0.875rem;
```
Comment (ahmetbozaci) [inline src/components/Book.scss:30]: - [x] For DRY CSS you should combine duplicate properties. 
Example, you can create class `.d-flex { display: flex}` and use this class in HTML file.
_This is just one example. Check your CSS file and if some CSS rules are the same please combine them also._
- [More information about DRY principles](https://www.freecodecamp.org/news/7-important-tips-for-writing-better-css/) 

- [Html and CSS best practices in curriculum](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md#:~:text=If%20you%20want%20to%20create%20a%20CSS%20rule%20for%20multiple%20elements%2C%20use%20a%20class%20attribute%20instead.%20For%20example%2C%20instead%20of%20using%20color%3A%20%233498db%3B%203%20times)

**Review by ahmetbozaci — CHANGES_REQUESTED** [review]
# ♻️♻️♻️ STATUS: Changes Required  ♻️♻️♻️


 @Said-laasri  ✋🏼 

#### Your project is great. You are almost done. However, there are some issues you should fix. Please fix them then your project will be approved.

___ 

### 💯  Highlights 💯
- Your git-flow is correct. ✔️
- You set up linter files and no errors. ✔️
- Your pull request is very professional. It has a meaningful title and summary. Keep it up.✔️
___


### ♻️ Request changes ♻️

- [x] _Check the comments under the review._

___

### Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
@Said-laasri 
⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ 

- Your commit messages should always start with capital letters and they should be an imperative tense. I see some of them are not like that. Instead of writing `Added style or Created the file` you should write `Add style or Create the file`
- ***I write it as optional. But this is a [general requirement](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md#:~:text=The%20commit%20message%20should%20be%20written%20in%20the%20imperative%20tense%20and%20capitalized) for every project. So you should pay attention to this in your next projects.***
- You can change them by using `git-rebase`. If you want to know how to change ı leave a few resources.
- [Video - How to change commit messages](https://www.loom.com/share/00771f369b7b47fc968c019ecf0d3ba6)
- [Article - How to create good commit messages](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8/)

⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ 
___

### Cheers and Happy coding! 💻 👏
almost


___

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification._

_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._


___

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)_

[GitHub](https://github.com/ahmetbozaci) | [LinkedIn](https://linkedin.com/in/ahmetbozaci) | [Twitter](https://twitter.com/ahmtbozaci)

_Reviewed by [Ahmet Bozacı ](https://github.com/ahmetbozaci)_

File: src/components/Book.jsx (hunk 1 above)
Comment (Said-laasri) [reply src/components/Book.jsx]: @ahmetbozaci   Thank you for reviewing my project, I will make the necessary corrections as suggested

File: src/components/Book.scss (hunk 2 above)
Comment (Said-laasri) [reply src/components/Book.scss]: @ahmetbozaci  Thank you for reviewing my project, I will make the necessary corrections as suggested

File: src/components/Book.scss (hunk 4 above)
Comment (Said-laasri) [reply src/components/Book.scss:30]: @ahmetbozaci   Thank you for reviewing my project, I will make the necessary corrections as suggested

File: src/components/Book.jsx [hunk 5]
```diff
@@ -23,22 +23,22 @@ function Book(props) {
...
-        <div className="book-info">
-          <h3>{category}</h3>
-          <h2>{title}</h2>
-          <h3>{author}</h3>
+    <div className="book-container d-flex">
+      <div className="book d-flex">
+        <div className="book-info d-flex">
+          <h3 className="title1 font-mont">{category}</h3>
+          <h2 className="title2 font-Roboto">{title}</h2>
+          <h3 className="title1 font-mont">{author}</h3>
```
Comment (mateo951) [inline src/components/Book.jsx:31]: Addressed ✔

**Review by mateo951 — APPROVED** [review]
### Hi @Said-laasri, 

Well done making the changes requested by your previous reviewer 🔥
Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀 
### Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Book.jsx (hunk 5 above)
Comment (Said-laasri) [reply src/components/Book.jsx:31]: thanks for review

**Comment by Said-laasri** [comment]
@mateo951  Thank you for reviewing my project.

**Pull request merged**